include/GafferUI/StandardConnectionGadget.h

```diff
@@ -117,6 +117,12 @@ class StandardConnectionGadget : public ConnectionGadget
 		bool m_dotPreview;
 		Imath::V3f m_dotPreviewLocation;
 
+		bool m_addingConnection;
+		Imath::V3f m_srcPosOrig;
```

Comment:
Do we need all four of these V3fs? It seems like two might be sufficient since there's only a new endpoint for the dragged connection - the source point is always shared. We make quite a lot of these gadgets, and they already have a fairly big footprint, so it would be nice to minimise it if possible...